### DISCUSSION TITLE
About Docker image

### DISCUSSION THREAD

**Discussion by shuncon** [ddiu8081/chatgpt-demo#75] [Q&A]
docker镜像呢

**Comment by quzard** [comment]
我编译了一个
```text
version: "3"

services:
  chatgpt:
    image: quzard/chatgpt-demo:latest
    container_name: chatgpt
    restart: always
    ports:
        - "3000:3000"
    volumes:
      - .env:/app/.env
 ```

**Reply by wzmtgggg** [reply]
docker在哪里配置自己的key啊

**Comment by Mircc** [comment]
大佬也是牛哇

**Comment by nevertoday** [comment]
真的是牛！

**Comment by dckill** [comment]
我也是正想干这个事,感谢

**Comment by Lau0x** [comment]
> 我编译了一个
> 
> ```
> version: "3"
> 
> services:
>   chatgpt:
>     image: quzard/chatgpt-demo:latest
>     container_name: chatgpt
>     restart: always
>     ports:
>         - "3000:3000"
>     volumes:
>       - .env:/app/.env
> ```
感谢！

**Reply by wzmtgggg** [reply]
docker怎么配置key啊

**Comment by Xueshen6** [comment]
> 

老哥这个配置https服务端，我用nginx将3000端口反向代理到443，怎么试都不成功，能不能支个招

**Comment by quzard** [comment]
> > 
> 
> 老哥这个配置https服务端，我用nginx将3000端口反向代理到443，怎么试都不成功，能不能支个招

有啥问题？我这没啥问题。是不是nginx配置了这个？

location ~ /\.(?!well-known) {
    deny all;
}

可能是这个规则导致的，可以注释了试试

**Comment by Xueshen6** [comment]
> > 老哥这个配置https服务端，我用nginx将3000端口反向代理到443，怎么试都不成功，能不能支个招
> 
> 有啥问题？我这没啥问题。是不是nginx配置了这个？
> 
> location ~ /.(?!well-known) { deny all; }
> 
> 可能是这个规则导致的，可以注释了试试



> > 老哥这个配置https服务端，我用nginx将3000端口反向代理到443，怎么试都不成功，能不能支个招
> 
> 有啥问题？我这没啥问题。是不是nginx配置了这个？
> 
> location ~ /.(?!well-known) { deny all; }
> 
> 可能是这个规则导致的，可以注释了试试


> > 老哥这个配置https服务端，我用nginx将3000端口反向代理到443，怎么试都不成功，能不能支个招
> 
> 有啥问题？我这没啥问题。是不是nginx配置了这个？
> 
> location ~ /.(?!well-known) { deny all; }
> 
> 可能是这个规则导致的，可以注释了试试

我是这么写的
server {
    listen 80 default;
    server_name _;
    return 301 https://$host$request_uri;
}
server {
    listen 443 ssl;
    server_name domain.com domain.com;
    ssl_certificate ssl/domain.com.crt;
    ssl_certificate_key ssl/domain.com.key;
    location / {
        proxy_pass http://127.0.0.1:3000;
    }
}
